tasks/README.md

```diff
@@ -1,6 +1,82 @@
 # gem5art tasks package
 
-This package contains a [Celery](http://www.celeryproject.org/) worker and tasks for running gem5.
+This package contains two parallel task libraries for running gem5 experiments.
+he actual gem5 experiment can be executed with the help of [Python multiprocessing support](https://docs.python.org/3/library/multiprocessing.html), [Celery](http://www.celeryproject.org/) or even without using any job manager (a job can be directly launched by calling `run()` function of gem5Run object).
 This package implicitly depends on the gem5art run package.
 
-See [gem5art](https://github.com/darchr/gem5art) repo for more details.
+Please cite the the [gem5art paper](https://arch.cs.ucdavis.edu/papers/2021-3-28-gem5art) when using the gem5art packages.
```

Comment:
extra "the"!

Reply:
Good find! Though, you missed it in the run and artifact readmes :laughing: I just copy-pasted this line each time.